utils/trace/trace_test.go

```diff
@@ -0,0 +1,74 @@
+package trace
+
+import (
+	"bytes"
+	"log"
+	"testing"
+
+	. "gopkg.in/check.v1"
```

Comment:
For new code, we should move away from `check.v1` as that dependency is no longer maintained.

Reply:
switched to stdlib testing